next/src/components/_global/informationComponent/informationComponent.tsx

```diff
@@ -0,0 +1,14 @@
+import styles from './informationComponent.module.scss';
+import { type Props } from './informationComponent.types';
+import Markdown from '@/components/ui/Markdown';
+
+export default function InformationComponent({ data: { heading, description } }: Props) {
+  return (
+    <section className={styles.informationComponent}>
+      <Markdown.h2>{heading}</Markdown.h2>
+      <div className={styles.markdownWrapper}>
+        <Markdown className={styles.markdown}>{description}</Markdown>
+      </div>
```

Comment:
Is the `markdownWrapper` is necessary? When you add `className` to the `<Markdown>` it will automatically add wrapper with that `className`

Reply:
The gap in Markdown, and between the Markdown is different. This is the first approach. The second approach would be defining styles like paddings and margins in styles